core/src/main/java/org/hibernate/ogm/hibernatecore/impl/OgmSessionFactoryImpl.java

```diff
@@ -91,6 +92,11 @@ public OgmSession getCurrentSession() throws HibernateException {
 		return new OgmSessionImpl( this, (EventSource) session );
 	}
 
+	@Override
+	public CriteriaBuilder getCriteriaBuilder() {
+		throw new NotSupportedException( "OGM-23", "Criteria queries are not supported yet" );
```

Comment:
I don't know if we should add a test for it

Reply:
We usually don't have tests for feature we don't support. I think it's OK to not add a new test for this.